bench/ssz.exs

```diff
@@ -34,3 +34,26 @@ Benchee.run(
   warmup: 2,
   time: 5
 )
+
+## Benchmark Merkleization 
+
+list = Stream.cycle([65_535]) |> Enum.take(316)
```

Comment:
Isn't this the same as 1..316 |> Enum.to_list()?

Reply:
Naa...this is [65_535, 65_535...., 65_535] of length 316

Reply:
ahh, okay.